detection/notebooks/11_exploring_hyperparameters_on_azureml.ipynb

```diff
@@ -0,0 +1,618 @@
+{
```

Comment:
<p>maybe we should log some of these params with scrapbook?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/ComputerVision/pull/303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Good point. I will add some of that once we add the notebook tests

Reply:
Added.